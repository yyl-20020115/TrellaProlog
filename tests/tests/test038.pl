:- initialization(main).

main :- write('foo
  bar
  baz'), nl.
