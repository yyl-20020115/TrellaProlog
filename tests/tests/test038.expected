Error: syntax error, unterminated quoted atom, tests/tests/test038.pl:3
